classic_ml/README.md

```diff
@@ -0,0 +1,80 @@
+# Classical ML Models
+
+This folder contains the script to run Traditional ML models from scikit-learn using onnx framework.
+
+Each model has been trained on specific dataset and it should be tested using only that specific dataset. For ex Regression models are using diabetes dataset and classification models are using iris dataset.
+
+Dataset can be downloaded from here:
+```
+https://ampereaimodelzoo.s3.eu-central-1.amazonaws.com/sklearn2onnx/*.csv
+
+```
+
+### Metrics
```

Comment:
What was the system under test? I guess Altra but how many cores etc?

Reply:
I used AltraMax.